backend/grants/admin.py

```diff
@@ -317,12 +310,59 @@ class GrantAdmin(ExportMixin, AdminUsersMixin, SearchUsersMixin):
     )
     user_fk = "user_id"
     actions = [
-        send_messages_to_plain,
         send_reply_emails,
         send_grant_reminder_to_waiting_for_confirmation,
         "delete_selected",
     ]
 
+    fieldsets = (
+        (
+            "Manage the Grant",
+            {
+                "fields": (
+                    "status",
+                    "approved_type",
+                    "country_type",
+                    "ticket_amount",
+                    "travel_amount",
+                    "accommodation_amount",
+                    "total_amount",
+                    "applicant_message",
+                    "applicant_reply_sent_at",
+                    "applicant_reply_deadline",
+                )
+            },
+        ),
+        (
+            "About the Applicant",
```

Comment:
same, the order seems off

when I read a grant I imagine I should see

The grant I am looking at -> Info about the person -> Actions to modify it